server/routes/updateGoal/updateGoalController.ts

```diff
@@ -41,10 +41,11 @@ export default class UpdateGoalController {
   submitUpdateGoalForm: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber, goalReference } = req.params
     const updateGoalForm: UpdateGoalForm = { ...req.body }
+    const updateStepForm: UpdateStepForm = { ...req.body }
```

Comment:
Not sure I follow this (or, it feels kinda weird) - how can 2 different variables of 2 different types be instantiated from the same data? (`req.body`)
```
    const updateGoalForm: UpdateGoalForm = { ...req.body }
    const updateStepForm: UpdateStepForm = { ...req.body }
```